### ISSUE TITLE
Fix issue with maps where it doesn't show pins when switching between facility list and facility

### ISSUE THREAD

**Issue by jonminter** [540co/epa-rfi#84]

**Comment by jonminter** [comment]
Was working locally but when deployed still see the issue

**Comment by patricknowlan** [comment]
I think i fixed this issue by adding an ng-if to the map that checks that the data has been loaded before rendering the map. In pull request #114. This might also be a duplicate of #108 which I closed. I noticed a new issue where the map does not show pins when the state changes based off of navigating screens using the browsers back button.

**Comment by patricknowlan** [comment]
Should we create a new issue or track it here?

**Comment by patricknowlan** [comment]
I think this is an issue with the state.

**Comment by patricknowlan** [comment]
Looks like this is partially fixed. It works for navigating back and forth between the facility list and facility page but the map will not work if you press back until you hit the dashboard page and select a new state. It will require the user to refresh their page once they land on the facility list view in order to view the map
